contracts/smart-account/interfaces/modules/ISecurityPolicyManagerPlugin.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {ISecurityPolicyPlugin} from "./ISecurityPolicyPlugin.sol";
+
+address constant SENTINEL_MODULE_ADDRESS = address(0x1);
+
+interface ISecurityPolicyManagerPluginEventsErrors {
+    event SecurityPolicyEnabled(address indexed sa, address indexed policy);
+    event SecurityPolicyDisabled(address indexed sa, address indexed policy);
+    event ModuleValidated(address indexed sa, address indexed module);
+
+    error SecurityPolicyAlreadyEnabled(address policy);
+    error SecurityPolicyAlreadyDisabled(address policy);
+    error InvalidSecurityPolicyAddress(address policy);
+    error InvalidPointerAddress(address pointer);
+    error ModuleInstallationFailed();
+    error EmptyPolicyList();
+    error ModuleIsNotAContract(address module);
+}
+
+/// @title Security Policy Manager Plugin
+/// @author @ankurdubey521
+/// @dev Execution Phase Plugin responsible for enforcing security policies during plugin installation on the smart contract wallet
+interface ISecurityPolicyManagerPlugin is
+    ISecurityPolicyManagerPluginEventsErrors
+{
+    /// @dev Enables the security policies for the smart contract wallet. Used during the setup process.
```

Comment:
I noticed you used the term "smart contract wallet". Should we stick with this or use "smart account" instead?